sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -129,7 +129,7 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
 
     checkAnswer(
       sql(s"SHOW TBLPROPERTIES parquet_tab1('my_key1')"),
-      Row("v1") :: Nil
+      Row("my_key1", "v1") :: Nil
```

Comment:
is it a reasonable behavior change?

Reply:
> is it a reasonable behavior change?

To be honest  I am confused that why in `ShowTablePropertiesCommand`  we don't show `key` if not specified 
but we will show always show `key`  in `ShowTableProperties`

Keep consistent I think is necessary.

Reply:
Yea we need to keep consistent. The question is which side we should follow. What's the behavior in hive?

Reply:
> Yea we need to keep consistent. The question is which side we should follow. What's the behavior in hive?

![image](https://user-images.githubusercontent.com/46485123/106471775-f58a5b00-64dc-11eb-9aaf-51bfa7e2d785.png)

![image](https://user-images.githubusercontent.com/46485123/106471914-181c7400-64dd-11eb-9aa7-fe045e3507ec.png)

Seems hive is same as ShowTablePropertiesCommand

Reply:
then shall we change the schema of the v2 command?

Reply:
But seems changed in hive current master branch https://github.com/apache/hive/blob/95c7d361549272c52dfcf637eb2a0360b79b9815/ql/src/java/org/apache/hadoop/hive/ql/ddl/table/ShowTablePropertiesOperation.java

Reply:
> then shall we change the schema of the v2 command?

IMO,  change  `ShowTablePropertiesCommand` is correct since hive's output schema is `prpt_name` and `prpt_value`, I think hive is wrong before.

Reply:
![image](https://user-images.githubusercontent.com/46485123/106477773-764c5580-64e3-11eb-8536-f2c62b54d5a8.png)

Add commit `keep same as 2.x hive` as a  backup and revert it.  Don't be confused about this

Reply:
> > then shall we change the schema of the v2 command?
> 
> IMO, change `ShowTablePropertiesCommand` is correct since hive's output schema is `prpt_name` and `prpt_value`, I think hive is wrong before.

Also ping @wangyum @beliefer @viirya Also hope your suggestion since I believe it's a hive's bug and we don't need to follow wrong behavior in hive 1.x and 2.x

Reply:
Changing the output schema looks fine to me. Could you describe this change in the PR description?

Reply:
> Changing the output schema looks fine to me. Could you describe this change in the PR description?

Yea, updated, Do we need to change migration guide too?

Reply:
Yea let's add an item in the migration guide.

Reply:
> Yea let's add an item in the migration guide.

Done.